cli/bin/dev

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
```

Comment:
@gimenete do we want to try clipanion? It's ES ready and a bit more typesafe for parsing the options and should have the same benefit as oclif for autogenerated help and testability

Reply:
I took a look to it but oclif supports many other things such as plugins, already built-in behaviours such the --json flag, generating binaries ready to be uploaded to homebrew, etc.